README.md

```diff
@@ -187,9 +189,30 @@ Linux users can view the docs by running:
 
     xdg-open docs/_build/html/index.html
 
+### Contributing to the Augur Docs
+
+Sphinx, coupled with [reStructuredText], can be tricky to learn. Here are a couple of pointers with
+linked resources to get you started.
+
+* Most Sphinx documents exist as [reStructuredText] (.rst) files.
+  
+* Many Sphinx reStructuredText files contain a [directive] to add relations between single files in
+  the documentation, known as a Table of Contents Tree ([TOC Tree]).
+
+* Human-readable augur and augur subcommand documentation is written using a Sphinx extension called
+  [sphinx-argparse].
+
```

Comment:
What do you think about linking to https://simonwillison.net/2018/Aug/25/restructuredtext/ as well?